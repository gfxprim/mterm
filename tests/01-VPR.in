3 3
\na
\e[ea
