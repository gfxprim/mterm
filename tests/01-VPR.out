 ---
|   |
|a  |
| a |
 ---
size 3x3 cursor 2x2
